code/modules/admin/view_variables/topic_basic.dm

```diff
@@ -77,6 +78,11 @@
 			target._AddElement(lst)
 		log_admin("[key_name(usr)] has added [result] [datumname] to [key_name(target)].")
 		message_admins("<span class='notice'>[key_name_admin(usr)] has added [result] [datumname] to [key_name_admin(target)].</span>")
+	if(href_list[VV_HK_MODIFY_GREYSCALE])
+		if(!check_rights(NONE))
+			return
```

Comment:
shouldn't we crash here?

Reply:
We usually don't crash when permission checks fail, if I remember right a message gets thrown out for admins if it fails anyway.

Reply:
Ah excuse me I thought this was a ahhhhhhhhhhhhhhh funny javascript rights bypass check, but players can view this panel with admin permission can't they